<launch>
   <node name="pipeline_vision" pkg="visualization" type="visual_pipe_plus.py" output="screen"/>
   <!-- <node name="decision_node" pkg="decision" type="main.py" output="screen"/> -->
   <!-- <node name="obj_send" pkg="rflysim_ros_pkg" type="send_test.py" /> -->
   <arg name="ip" default="192.168.199.140" />
   <arg name="mav_x" default="-188" />
   <arg name="mav_y" default="-192" />
   <arg name="mav_z" default="0" />
   <arg name="mav_yaw" default="85" />
   <node pkg="rflysim_ros_pkg" type="Rflysim_objwithsize.py" name="obj_control" output="screen">
        <param name="ip" value="$(arg ip)"/>
        <param name="mav_x" value="$(arg mav_x)"/>
        <param name="mav_y" value="$(arg mav_y)" />
        <param name="mav_z" value="$(arg mav_z)" />
        <param name="mav_yaw" value="$(arg mav_yaw)" />
   </node>
   <!-- <node name="obj_control" pkg="rflysim_ros_pkg" type="Rflysim_obj.py" output="screen" /> -->
   <!-- <node name="obj_send" pkg="rflysim_ros_pkg" type="send_test.py" /> -->
</launch>